plugins/outputs/stackdriver/stackdriver.go

```diff
@@ -39,18 +41,20 @@ type Stackdriver struct {
 	TagsAsResourceLabels []string          `toml:"tags_as_resource_label"`
 	MetricCounter        []string          `toml:"metric_counter"`
 	MetricGauge          []string          `toml:"metric_gauge"`
+	MetricHistogram      []string          `toml:"metric_histogram"`
 	Log                  telegraf.Logger   `toml:"-"`
 
-	client        *monitoring.MetricClient
-	counterCache  *counterCache
-	filterCounter filter.Filter
-	filterGauge   filter.Filter
+	client          *monitoring.MetricClient
+	counterCache    *counterCache
+	filterCounter   filter.Filter
+	filterGauge     filter.Filter
+	fitlerHistogram filter.Filter
 }
 
 const (
 	// QuotaLabelsPerMetricDescriptor is the limit
 	// to labels (tags) per metric descriptor.
-	QuotaLabelsPerMetricDescriptor = 10
+	QuotaLabelsPerMetricDescriptor = 30
```

Comment:
Is there any cite-able source for this number? If so we should probably put a link in here...

Reply:
PR updated with link to: https://cloud.google.com/monitoring/quotas#custom_metrics_quotas